R/method-differentialAbundance.R

```diff
@@ -53,34 +53,53 @@ setMethod("differentialAbundance", signature("AbsoluteAbundanceData"), function(
       veupathUtils::logWithTime("Replaced NAs with 0", verbose)
     }
 
+    # Err if the class of groupA does not match that of groupB
+    if (!identical(class(groupA), class(groupB))) {
+      stop("groupA and groupB must either both be character vectors or both be BinLists.")
+    }
+
+    # Determine if comparison variable is numeric.
+    isNumericComparisonVar <- identical(class(groupA)[1], "BinList")
```

Comment:
the heuristics make me uncomfortable. this is the kind of thing that becomes a bug later..